playground/nextjs/package.json

```diff
@@ -11,8 +11,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@clerk/nextjs": "beta",
-    "@clerk/themes": "beta",
+    "@clerk/backend": "file:.yalc/@clerk/backend",
+    "@clerk/clerk-react": "file:.yalc/@clerk/clerk-react",
+    "@clerk/clerk-sdk-node": "file:.yalc/@clerk/clerk-sdk-node",
+    "@clerk/nextjs": "file:.yalc/@clerk/nextjs",
+    "@clerk/shared": "file:.yalc/@clerk/shared",
+    "@clerk/themes": "file:.yalc/@clerk/themes",
+    "@clerk/types": "file:.yalc/@clerk/types",
```

Comment:
Can we drop these changes ?

Reply:
ahh, and I thought I didn't commit this again